templates/js/main.js

```diff
@@ -5,7 +5,7 @@
       return el.className = el.className.replace(' dn', '');
     }
 
-    el.className = el.className + ' dn'.trim();
+    (el.className = el.className + ' dn').trim();
```

Comment:
Brackets in wrong place
(el.className = el.className + ' dn').trim();
should be
el.className = (el.className + ' dn').trim();